File: wed 17 4 2024/six.py
def verify_sw_tooltip(actualSWVersion, expSWVersion):
    result = False
    try:
        i = 0
        swVersion = actualSWVersion.split('-')[0].split('.')
        expVersion = expSWVersion.split('-')[0].split('.')
        for version in swVersion:
            if version > expVersion[i]:
                result = True
                break
            i += 1
    except Exception:
        print(f"Something went wrong in the method: verify_sw_tooltip")
        print(f"Exception caught: {str(sys.exc_info()[1])}\nError in method verify_sw_tooltip")
        print(f"Exception backtrace:\n {'\n'.join(traceback.format_exception(*sys.exc_info()))}")
    finally:
        return result

File: wed 17 4 2024/six.rb
 def self.verify_sw_tooltip(actualSWVersion, expSWVersion)
      expVersion      = []
      result          = false
      i               = 0
      swVersion       = actualSWVersion.split('-')[0].split('.')
      expVersion      = expSWVersion.split('-')[0].split('.')
      swVersion.each do |version|
         if (version > expVersion[i])
            result = true
            break
         end
         i += 1
      end
   rescue
      @logger.log_message("Something went wrong in the method : #{__method__}", LogUtilities::ERROR, false)
      @logger.log_message("Exception caught: #{$!.inspect}\nError in method #{__method__}", LogUtilities::ERROR)
      @logger.log_message("Exception backtrace:\n #{$!.backtrace.join("\n")}", LogUtilities::DEBUG, false)
   ensure
      return result
   end